app/_assets/stylesheets/base.less

```diff
@@ -13,7 +13,7 @@ body {
   -webkit-font-smoothing: antialiased;
 
   .container {
-    max-width: 1020px;
```

Comment:
With table reformatting (yay!), do we still need to increase this width? This seems like a pretty major change to the website, so if we can get away without doing it, that'd be great. Otherwise, I'm afraid we'd need the approval of someone from the design team before going ahead with this change.

Reply:
Yeah I agree - I thought it might help make sure this doesn't happen on any other pages but it doesn't look like it is an issue with the reformatting.